Packs/MicrosoftEntraID/ReleaseNotes/1_0_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Microsoft Entra ID Parsing Rule
+
+- Updated the Microsoft Entra ID Parsing Rule parsing rule to support Azure DevOps log ingestion via Azure Event hub.
```

Comment:
Updated the Microsoft Entra ID Parsing Rule to support Azure DevOps log ingestion via Azure Event hub.  (removed second 'parsing rule')